src/solc_0.8/polygon/LiquidityMining/PolygonLandWeightedSANDRewardPool.sol

```diff
@@ -200,6 +200,8 @@ contract PolygonLandWeightedSANDRewardPool is PolygonLPTokenWrapper, IRewardDist
             _rewardToken.safeTransfer(msg.sender, reward);
             emit RewardPaid(msg.sender, reward);
         }
+
+        updateContribution(msg.sender);
```

Comment:
Could you change every msg.sender by _msgSender()

Reply:
2771 n'est pas implémenté sur ce contrat